.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "pointgrey_camera_driver"]
+	path = pointgrey_camera_driver
+	url = https://github.com/ros-drivers/pointgrey_camera_driver.git
+[submodule "rj-ros-common"]
+	path = rj-ros-common
```

Comment:
can we put both submodules in a folder called third_party? So third_party/rj-ros-common and third_party/pointgrey_camera_driver. This is common practice and it will make sure no one gets these repos confused with our packages